docs/docs_source/build_test_ship/settings.rst

```diff
@@ -0,0 +1,105 @@
+
+..  _settings:
+
+Microsoft-configured settings
+=============================
+
+..  spelling::
+
+    subdomain
+    wopi
+    qa
+
+Most Office Online behavior is configurable based on properties provided in :ref:`CheckFileInfo`. However, there are
+some Office Online settings that must be changed by Microsoft.
```

Comment:
I think this page would benefit from a quick list of all of the settings, with a link to the individual section below that describes it in more detail.